ext/TensorKitChainRulesCoreExt.jl

```diff
@@ -536,9 +536,8 @@ function qr_pullback!(ΔA::AbstractMatrix, Q::AbstractMatrix, R::AbstractMatrix,
                       atol::Real=0,
                       rtol::Real=atol > 0 ? 0 : eps(real(eltype(R)))^(3 / 4))
     Rd = view(R, diagind(R))
-    p = let tol = atol > 0 ? atol : rtol * maximum(abs, Rd)
-        findlast(x -> abs(x) >= tol, Rd)
-    end
+    tol = atol > 0 ? atol : rtol * maximum(abs, Rd)
+    p = findlast(x -> abs(x) >= tol, Rd)
```

Comment:
You can avoid the closure problem by using `findlast(Base.Fix2(>=, tol) ∘ abs, Rd`

Reply:
Thanks!